tests/manager/models/conftest.py

```diff
@@ -0,0 +1,65 @@
+# conftest.py
+from uuid import UUID
+
+import pytest
+from graphene import Schema
+from graphene.test import Client
+
+from ai.backend.manager.models.gql import GraphQueryContext, Mutations, Queries
+
+
+@pytest.fixture(scope="module", autouse=True)
+def client() -> Client:
+    return Client(Schema(query=Queries, mutation=Mutations, auto_camelcase=False))
+
+
+@pytest.fixture(scope="function")
+def get_base_context():  # noqa: F811
+    def _get_base_context(**overrides):
+        """
+        default_params is used to bypass the decorators.
+        """
+        default_params = {
+            "schema": None,
+            "dataloader_manager": None,
+            "local_config": None,
+            "shared_config": None,
+            "etcd": None,
+            "user": {
+                "role": "user",
+                "email": "example@lablup.com",
+                "uuid": UUID("f38dea23-50fa-42a0-b5ae-338f5f4693f4"),
+                "domain_name": "default",
+            },
+            "access_key": "AKIAIOSFODNN7EXAMPLE",
+            "db": None,
+            "redis_stat": None,
+            "redis_image": None,
+            "redis_live": None,
+            "manager_status": None,
+            "known_slot_types": None,
+            "background_task_manager": None,
+            "storage_manager": None,
+            "registry": None,
+            "idle_checker_host": None,
+        }
+        updated_params = deep_update(default_params, overrides)
+        return GraphQueryContext(**updated_params)
+
+    return _get_base_context
+
+
+def deep_update(original: dict, updates: dict) -> dict:
```

Comment:
We already have `ai.backend.agent.utils.update_nested_dict()`. Maybe we could move it to `ai.backend.common.utils`.
I think it is also better *not* to return the original value when there is a side-effect to mutate the given argument (`original`).

Reply:
I make some changes:

1. Moved `ai.backend.agent.utils.update_nested_dict()` -> `ai.backend.common.utils`
2. Relocated the test code for `update_nested_dict()`from **agent** to the **common** module.
3. Removed `deep_update`and replace its usage with `update_nested_dict` function